test/legacy/roles/scaleway_s3/tasks/aws_s3.yml

```diff
@@ -0,0 +1,19 @@
+- name: Create a local file
+  tempfile:
+    state: file
+    suffix: temp
+  delegate_to: localhost
+  register: tempfile
+
+- include: aws_s3_create.yml
+
+- include: aws_s3_put.yml
+- include: aws_s3_get.yml
+
+- include: aws_s3_getstr.yml
```

Comment:
There is no difference between `aws_s3_get.yml` and `aws_s3_getstr.yml`, keep only one task file.